### ISSUE TITLE
Issue with filtering on members page

### ISSUE THREAD

**Issue by beaugunderson** [OpenHumans/open-humans#662]
![screenshot](https://i.imgur.com/bVuvWoe.png)

```
member-list.js:7 Uncaught TypeError: Cannot read property 'defaults' of undefined
    at Object.r.3.jquery (member-list.js:7)
    at i (_prelude.js:1)
    at e (_prelude.js:1)
    at _prelude.js:1
```

The line in question:

```
$.fn.select2.defaults.set('theme', 'bootstrap');
```

Looks like an issue with `select2`, not immediately apparent. If jQuery or select2 were updated to a new version recently that might be the cause?

**Comment by BenjaminHCCarr** [comment]
`jquery` got bumped for security reasons

**Comment by beaugunderson** [comment]
@BenjaminHCCarr last change i see to jquery was the one i made >1 year ago, are you referring to something else?

**Comment by BenjaminHCCarr** [comment]
@beaugunderson I thought @madprime bumped it since then, there was a remote execution problem, but I must be mistaken

**Comment by BenjaminHCCarr** [comment]
I found my PR it was in *Genevieve madprime/genevieve/pull/7*

**Comment by beaugunderson** [comment]
ah i see :)

**Comment by beaugunderson** [comment]
fix for this is in #675 ✨

**Comment by madprime** [comment]
Deployed and fixed now – thank you @beaugunderson :tada: :D
